platform/darwin/src/MGLGeometry_Private.h

```diff
@@ -34,6 +34,14 @@ NS_INLINE mbgl::LatLngBounds MGLLatLngBoundsFromCoordinateBounds(MGLCoordinateBo
                                     MGLLatLngFromLocationCoordinate2D(coordinateBounds.ne));
 }
 
+NS_INLINE void MGLLocationCoordinatesFromCoordinateBounds(MGLCoordinateBounds coordinateBounds, CLLocationCoordinate2D *coordinates) {
+    assert(sizeof(coordinates) == 4 * sizeof(CLLocationCoordinate2D));
```

Comment:
My bad, you want NSCAssert, because we're in a C function that has no access to self.